### DISCUSSION TITLE
Linking GMP and MPFR on Windows with Visual Studio

### DISCUSSION THREAD

**Discussion by canberkirimagzi** [CGAL/cgal#8491] [Q&A] [answered]

**Comment by janetournois** [comment, marked as answer]
You can find [here](https://doc.cgal.org/latest/Manual/windows.html#sec-installing-with-vcpkg) the documentation of how to compile CGAL examples with dependencies built from vcpkg

**Comment by canberkirimagzi** [comment]
> You can find [here](https://doc.cgal.org/latest/Manual/windows.html#sec-installing-with-vcpkg) the documentation of how to compile CGAL examples with dependencies built from vcpkg

I thought your answer was spam and that's why I gave a thumbs-down. I can't jugde if it is a spam or not
